tests/PySys/call_tedge_-V/pysystest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pysystest type="auto">
+
+  <description>
+	<title></title>
```

Comment:
Having a title is a really good "pysys practice" which will really help during the pysys report generation.

Reply:
I added the titles here, lets have a discussion about how the best procedure to generate report